tools/storyteller/storyteller.proto

```diff
@@ -0,0 +1,29 @@
+syntax = "proto3";
+package tools.storyteller;
+
+option java_package = "tools.storyteller";
+option java_outer_classname = "Protos";
+
+message Story {
+  string id = 1;
+  int64 started_ms = 2;
+  string oneliner = 3;
+  string project_id = 4;
+  bool is_shared = 5;
+  string author = 6;
+}
+
+message Track {
+  string id = 1;
+  int64 started_ms = 2;
```

Comment:
I find it better to have `story_id` second. That way, when we add yet more fields, it's not somewhere in the middle.

Reply:
Thanks!